docs/content/tutorials/nginx-ingress-istio.md

```diff
@@ -0,0 +1,284 @@
+---
+title: NGINX Ingress Controller and Istio Service Mesh
+description: |
+  Use NGINX Ingress Controller with Istio Service Mesh.
+weight: 1800
+doctypes: ["concept"]
+toc: true
+docs: "DOCS-889"
+---
+
+NGINX Ingress Controller can now be used as the ingress controller for applications that are running inside an Istio service mesh. This allows to continue using the advanced capabilities that NGINX IC provides on Istio-based environments without resorting to any workarounds. 
+
+
+Prior to 1.11 release, a configuration below would send two host headers to the backend; 
+
+```yaml
+apiVersion: k8s.nginx.org/v1
+kind: VirtualServer
+metadata:
+  name: foo 
+spec:
+  host: foo.example.com
+  upstreams:
+  - name: foo
+    port: 8080
+    service: backend-svc
+  routes:
+  - path: "/"
+    action:
+      proxy:
+        upstream: foo
+        requestHeaders:
+          set:
+          - name: Host
+            value: bar.example.com
+```
+
+In 1.11 release, NGINX Ingress controller will only send one host header, depending on how you configure Ingress. By default NGINX Ingress Controller will send `proxy_set_header $host`. If Ingress has been configured with `requestHeaders` per the above example, this ensures that only one set of headers will be sent to the upstream server. In short, by setting `action-proxy-requestHeaders` in the `VirtualServer` CRD, NGINX Ingress will only send the specified hears that have been defined.    

```

Comment:
Like above.  I am not following how this is relevant to a tutorial on how to implement this or why.  It appears to be out of a blog announcement.

Reply:
I think, my original idea was to explain under the hood what is happening. The use of `action-proxy-request-headers` allows the user to specify which headers to send to the `upstreams`. In this case, it would be the istio sidecar that is expecting certain headers.